web/src/components/Navigation/TopNavLinkButton.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from "next/link";
+import { ReactNode } from "react";
+
+import IconButton from "../Inputs/IconButton";
+
+interface Props {
+  href: string;
+  className?: string;
+  children?: ReactNode;
+}
+
+export default function TopNavLinkButton({ href, className, children }: Props) {
+  return (
+    <Link href={href}>
+      <IconButton
+        aria-label={`Navigate to ${href.replace("/", "")}`}
```

Comment:
I think there must be a better solution for making a more readable area-label. At least we could replace with spaces instead of just stripping slashes, no?

Reply:
Oh wait just realizes this only replaces the initial slash. Even so, still curious as to whether you think this is best way to do this.

Reply:
We can pass a variable with the name but is that really necessary in this case considering that href will always be a readable name for the navigation ("feed", "create", "profile")

Reply:
That makes sense. I was just thinking what would happen if the link was longer.